wpt-import-stats.py

```diff
@@ -112,14 +112,15 @@ def get_latencies(imports, prs):
 
         index_found = binary_search_import(merge_commit, imports)
         import_found = imports[index_found]
-        previous_import = imports[index_found - 1]
+        previous_import = imports[index_found - 1] if index_found > 0 else None
         # Check if I get my binary search right :)
         assert _compare_commits(merge_commit, import_found.wpt_sha) <= 0, \
             "PR merge point {} after import {}".format(
             merge_commit, import_found)
-        assert _compare_commits(merge_commit, previous_import.wpt_sha) > 0, \
-            "PR merge point {} before the previous import {}".format(
-            merge_commit, previous_import)
+        if previous_import != None:
+            assert _compare_commits(merge_commit, previous_import.wpt_sha) > 0, \
+                "PR merge point {} before the previous import {}".format(
+                merge_commit, previous_import)
```

Comment:
This makes sense. Previously due to some coincidences (cutoff date, and API result pagination), it never happened.